weblate/trans/tasks.py

```diff
@@ -153,22 +156,19 @@ def commit_pending(
     else:
         components = Component.objects.filter(translation__pk__in=pks)
 
-    # All components with pending units
+    hours_expr = F("commit_pending_age") if hours is None else Value(hours)
+    if using_postgresql():
```

Comment:
Type annotate prior to first use:

```suggestion
    age_cutoff: Expression
    if using_postgresql():
```